src/ProofOfConcepts/TAndCModal/tAndCModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { VoidFunctionComponent } from "react";
+import { Modal, ModalVariant, Button } from "@patternfly/react-core";
+import { Flex, FlexItem } from "@patternfly/react-core";
+
+export class SmallModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false,
+    };
+    this.handleModalToggle = () => {
+      this.setState(({ isModalOpen }) => ({
+        isModalOpen: !isModalOpen,
+      }));
+    };
+  }
+
+  render() {
+    const { isModalOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Button variant="primary" onClick={this.handleModalToggle}>
+          Show small modal
+        </Button>
+        <Modal
+          id="modalTerms"
+          variant={ModalVariant.small}
+          title="Terms and Conditions"
```

Comment:
Any text we display belongs in a json file for translation. This text defined by the title, as well as other text strings provided below, should be moved to a json file.

As far as which json file and how they should be organized is something I'm hoping another team member can provide guidance on. 😜

Reply:
In the PoC is ok to have the copy like this, we'll make the translation files when we'll prepare the PoC for production.

Reply:
Who could I ask about which json file?

Reply:
Oh okay, I will go forward with the other changes.